webview-ui/src/index.css

```diff
@@ -45,3 +45,23 @@
   background-color: rgba(255, 255, 255, 0.5);
   color: brown;
 }
+
+.pf-chatbot__message-text {
+  word-break: break-word;
+}
+.pf-chatbot__message-text > details > summary {
+  cursor: pointer;
+}
+.pf-chatbot {
+  background-color: var(--vscode-editor-background) !important;
+}
+.pf-chatbot--embedded {
+  border: 1px solid var(--vscode-editor-foreground) !important;
+  border-radius: 20px;
```

Comment:
I'm not quite sold on the border here.  Turning it off looks ok (see the screenshot for the width changes).

If we want to keep it, I suggest using the color `--vscode-textSeparator-foreground` instead of the `--vscode-editor-foreground` so it isn't so bright/prominent.